### DISCUSSION TITLE
Add support to specify --no-check-lock option for composer_normalize task

### DISCUSSION THREAD

**Discussion by RoSk0** [phpro/grumphp#945] [Ideas]
| Q               | A
| --------------- | ---
| Version         | `GrumPHP 1.5.0`
| Bug?            | no
| New feature?    | yes
| Question?       | no
| Documentation?  | no
| Related tickets | #595 , #596 , #645 

composer-normalize introduced `--no-check-lock ` option in [2.7.0](https://github.com/ergebnis/composer-normalize/releases/tag/2.7.0) release back in August  2020. 

This option is very useful when you work on a  library project and you only concerned about the `composer.json` file but not that much about the lock file that only exists in the development environment for the library itself.

It would be awesome to see it available via configuration for the `composer_normalize` task.

**Comment by veewee** [comment]
I'dd surely accept a PR for that. Are you willing to give it a try?

**Reply by RoSk0** [reply]
Sure thing! 

https://github.com/phpro/grumphp/pull/947
